source/lib-intro.tex

```diff
@@ -150,9 +150,9 @@
 if it raises a floating-point exception other than \tcode{FE_INEXACT}.
 The semantics of a call to a C standard library function
 evaluated as a core constant expression
-are those specified in Annex F of the C standard
+are those specified in \IsoC{}, Annex F
 \begin{footnote}
-See also ISO/IEC 9899:2018 section 7.6.
+See also \IsoC{}, 7.6.
```

Comment:
Should we add "subclause" before "7.6" ?

Reply:
Not according to the House Style examples of citing specific elements, I think.

Reply:
"""
Use the order of “document number:date, element” when referring to a specific element:

“…in accordance with ISO 12345:2018, 4.3.9.”
“…in accordance with ISO/IEC TR 6789:2020, Table 3.”
“…in accordance with ISO 12345:2018/Amd 1:2019, Figure C.1.”
"""

Only things like "Table", "Figure", or "Annex" (and presumably "Clause") get a name.